### PR TITLE
Adding Header and Navigation

### LINKED ISSUE #3: Develop Header with Navigation
#### Description
The task involves creating the website header based on the design provided in Figma. Additionally, implement navigation links that smoothly scroll the page to different sections upon clicking.

#### Tasks
- [x] Review the Figma design for the header component.
- [x] Create HTML/CSS structure for the header.
- [x] Implement navigation links to relevant sections of the site.
- [x] Test navigation functionality and smooth scrolling.
- [x] Ensure responsiveness and compatibility across devices.

Screen Shot 2023-12-05 at 9 46 56 PM

### PR DESCRIPTION
**Pull request by elimanzo**
Added header+navigation+css stylles
Closes #3 
Web:
webHeader

Mobile:
mobileHeader

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="navbar">
+          <div class="logo">
+            <li><a href="/">John Doe</a></li>
+          </div>
+          <div class="nav-links">
+            <li><a href="#projects">Projects</a></li>
```
Comment (alexrdrgz) [inline index.html:20]: So the best practice for a navigation would be the following structure

```
<nav>
  <ul>
    <li><a></a></li>
    <li><a></a></li>
  </ul>
</nav>
```

Comment (alexrdrgz) [reply index.html:20]: you could add another container in it if needing for spacing or flex box or something like that but the general structure is best like this, especially for screen readers

**Review by alexrdrgz — CHANGES_REQUESTED** [review]
Awesome edits this is looking real good, I left an example of how to structure a navigation. Other than that everythings looking good, i like that your using css embedded targetting as well :)

**Comment by joavelar** [comment]
This is looks good to me.

**Review by alexrdrgz — APPROVED** [review]
Looks good! Nice refactor on the nav

**Pull request merged**